src/components/EventDetails.spec.tsx

```diff
@@ -1,8 +1,8 @@
 import EventDetails from './EventDetails';
 import { mountWithProviders } from '../utils/testing';
-import { ZetkinEvent } from '../interfaces/ZetkinEvent';
+import { ZetkinEvent } from '../types/zetkin';
```

Comment:
These three interfaces from the same module should be imported in a single `import` statement, IMO.